app/va/vetext/client.py

```diff
@@ -50,24 +50,37 @@ def send_push_notification(self, mobile_app: str, template_id: str, icn: str, pe
             response.raise_for_status()
         except requests.exceptions.ReadTimeout:
             # Discussion with VEText: read timeouts are still processed, so are marking this as good
-            self.logger.warning('ReadTimeout exceptions are still processed, returning 201')
+            self.logger.warning('ReadTimeout raised sending push notification - Still processed, returning 201')
             # Logging as error.read_timeout so we can easily track it
             self.statsd.incr(f"{self.STATSD_KEY}.error.read_timeout")
+        except requests.exceptions.ConnectTimeout as e:
+            self.logger.warning('ConnectTimeout raised sending push notification - Retrying')
+            # Logging as error.read_timeout so we can easily track it
+            self.statsd.incr(f"{self.STATSD_KEY}.error.connection_timeout")
+            raise VETextRetryableException from e
         except requests.HTTPError as e:
-            self.logger.exception(e)
+            self.logger.warning('HTTPError raised sending push notification')
             self.statsd.incr(f"{self.STATSD_KEY}.error.{e.response.status_code}")
             if e.response.status_code in [429, 500, 502, 503, 504]:
+                self.logger.warning('Retryable exception raised with status code: %s', e.response.status_code)
                 raise VETextRetryableException from e
                 # TODO: add retries?
```

Comment:
Is this comment still relevant?

Reply:
Yes, because it tells us what the particular status code is. That way if VEText is having issues we can tell them what we're seeing on our end.

Reply:
the TODO comment doesn't tell them what we are seeing, it asks if we need to add retries - do we still need to add retries here?

Reply:
The TODOs can go. They were put in by the old team and we have it on our roadmap to rework PUSH Notifications entirely.

Reply:
TODOs removed.